storage/pdb/src/main/java/org/openscience/cdk/templates/AminoAcids.java

```diff
@@ -153,7 +153,7 @@
                             aminoAcid.setProperty(RESIDUE_NAME_SHORT, ac.getProperty(dictRef));
                         } else if (dictRef.getType().equals("pdb:id")) {
                             aminoAcid.setProperty(ID, ac.getProperty(dictRef));
-                            LOGGER.debug("Set AA ID to: ", ac.getProperty(dictRef));
+                            LOGGER.debug("Set AA ID to: " + ac.getProperty(dictRef));
```

Comment:
This is an unneeded change. Not a blocker. I am not sure if it still applies, but it removes the need to concatenate strings when not doing the actual debug info output.